torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -494,12 +494,12 @@ def ReplaceMoreTtManually(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
                 # aten.expand and ttnn.repeat has different meaning for their `shape` argument
                 # aten.expand: the desired output shape, where respective singleton dims are broadcasted
                 # ttnn.repeat: the number of times to repeat a respective singleton dim
-                input_tensor_shape = args[0].meta["val"].size()
+                operand_shape = args[0].meta["val"].size()
```

Comment:
I think this is just a variable rename?

Reply:
Thanks for spotting this.  It was an accident introduced by "Replace All".  I just reverted unrelated changes.